tools/test-drive/src/main.rs

```diff
@@ -0,0 +1,88 @@
+//! Test that users are able to inspec the MIR body of functions and types
+
+#![feature(rustc_private)]
+#![feature(assert_matches)]
+#![feature(result_option_inspect)]
+
+mod sanity_checks;
+
+extern crate rustc_middle;
```

Comment:
You still need nightly for these APIs

Reply:
We need nightly for sure. The github workers come with rustup, so `cargo build` download the latest nightly.

See https://github.com/rust-lang/project-stable-mir/pull/25/checks#step:3:10